portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,39 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+// photos data
+import photos_data from './photos-data.js';
+
 /**
- * Adds a random greeting to the page.
+ * Create photo component with caption 
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function createPhoto(photo) {
+    let component = [
```

Comment:
[Use const unless the variable needs to be reassigned](https://google.github.io/styleguide/jsguide.html#features-use-const-and-let)